tests/timeseries/test_sort_timestamps_monotonically.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import pandas as pd
+from random import randint
+from janitor.timeseries import sort_timestamps_monotonically
+
+
+# Random data for testing
+@pytest.fixture
+def timeseries_dataframe() -> pd.DataFrame:
+    """
+    Returns a time series dataframe
+    """
+    ts_index = pd.date_range("1/1/2019", periods=1000, freq="1H")
+    v1 = [randint(1, 2000) for i in range(1000)]
+    test_df = pd.DataFrame({"v1": v1}, index=ts_index)
+    return test_df
+
+
+@pytest.mark.timeseries
+def test_sort_timestamps_monotonically(timeseries_dataframe):
+    """Test that sort_timestamps_monotonically works as expected."""
+    from sklearn.utils import shuffle
```

Comment:
This shouldn't be necessary; pyjanitor has a `.shuffle()` class method that can be called directly.